zingolib/src/lightclient.rs

```diff
@@ -245,6 +245,27 @@ impl LightClient {
         };
         LightClient::read_wallet_from_buffer(config, BufReader::new(File::open(wallet_path)?))
     }
+
```

Comment:
Let's add a comment here that explains how this fn ensures that the witness tree is..  uh...   bound to the wallet last scanned height witness tree?

Reply:
I don't understand what you mean